rpm/qtdocgallery.spec

```diff
@@ -1,6 +1,6 @@
 Name:       qt5-qtdocgallery
 Summary:    Qt document gallery optional module
-Version:    5.0.0
+Version:    5.2.0
 Release:    1
 Group:      System/Libraries
```

Comment:
I guess it also can be dropped.

Reply:
Adjusted this and above and changes also the url to this repo